templates/tab/js/m365/src/components/sample/AzureFunctions.jsx

```diff
@@ -1,21 +1,20 @@
 import React from "react";
 import { Button, Loader } from "@fluentui/react-northstar";
 import { useData } from "./lib/useData";
-import * as axios from "axios";
-import { TeamsFx } from "@microsoft/teamsfx";
+import { BearerTokenAuthProvider, createApiClient, TeamsFx } from "@microsoft/teamsfx";
```

Comment:
M365 app uses teamsfx 2.x. Are these APIs included in teamsfx 2.x?

Reply:
Yes, have tested this template locally.